### PR TITLE
adding language specific assets

### LINKED ISSUE #122: Gobstones new GBB boards are not being properly displayed

### PR DESCRIPTION
**Pull request by luchotc**
Resolves #122 

Should i add editor assets too?

Depends on: https://github.com/mumuki/mumuki-bibliotheca-api/pull/95

### CODE REVIEW

File: src/scripts/config/http.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
   .module('editor')
   .config(function ($httpProvider) {
+
     $httpProvider.defaults.withCredentials = true;
+
+    $httpProvider.interceptors.push(function($q) {
+      return {
+        'request': function(config) {
+          //CORS doesn't allow using Allow-Origin: * with credentials.
+          if(config.url.indexOf('.runners.mumuki.io/assets/') !== -1){
```
Comment (fedescarpa) [inline src/scripts/config/http.js]: we must to refactor this, the hardcoded url is burning my eyes :stuck_out_tongue:

**Comment by luchotc** [comment]
As i'm using link(rel="import") instead of ng-include it's ok to make the request withCredentials=true but i made the change anyway because i agree that it should be false by default and only set it to true when using our API.

**Comment by flbulgarelli** [comment]
@luchotc is this ready to merge?

**Comment by luchotc** [comment]
@flbulgarelli  Yeah, i was waiting for Fede's review, but the idea of passing the credentials only in our API's request its his, so i think he's happy with it!

**Pull request merged**